aiida/cmdline/commands/node.py

```diff
@@ -303,6 +306,60 @@ def print_node_info(self, node, print_groups=False):
                     print gr_specs
 
 
+class _Delete(VerdiCommand):
+    def run(self, *args):
+        """
+        Deletes a node and it's inferred data provenance
+        :raise ValueError: if no valid pk or uuid is given.
+        """
+        import argparse
+        from aiida.common.exceptions import NotExistent
+        from aiida.utils.delete_nodes import delete_nodes
+
+        parser = argparse.ArgumentParser(
+            prog=self.get_full_command_name(),
+            description='delete a node and everything that originated from that node')
+        parser.add_argument('pks', type=int, default=None, nargs="*", help="ID of the nodes to delete.")
+        parser.add_argument('-u', '--uuids', default=None, nargs='*', help='The uuid of the nodes to delete')
+        parser.add_argument('-c', '--follow-calls',
+            help='follow the call links downwards when deleting. If a node is a WorkCalculation, will delete everything it called',
+            action='store_true')
+        parser.add_argument('-r', '--follow-returns',
+            help='follow the return-links downwards when deleting. If a node is a WorkCalculation, will delete everything it returned',
+            action='store_true')
+        parser.add_argument('-n', '--dry-run', help='dry run, does not delete', 
+            action='store_true')
+        # Commenting this option for now
+        # parser.add_argument('-f', '--force', help='force deletion, disables final user confirmation', action='store_true')
+        parser.add_argument('-v', '--verbosity', help='verbosity level', action='count', default=1)
```

Comment:
for the verbosity level: not sure it's very clear like this...

Reply:
This has been addressed in 3fcab81